### PR TITLE
Implement Bridge

### PR DESCRIPTION
**Pull request by jparisu**
Signed-off-by: jparisu <javierparis@eprosima.com>

### CODE REVIEW

File: include/databroker/communication/Bridge.hpp [hunk 1]
```diff
@@ -28,29 +30,86 @@ namespace eprosima {
...
+     *
+     * In case any inside \c Track creation fails it wil throw a \c InitializationException
+     */
     Bridge(
             const RealTopic& topic,
-            std::shared_ptr<ParticipantDatabase> participant_database);
+            std::shared_ptr<ParticipantDatabase> participant_database,
+            bool enable = false);
 
+    //! Destroyer
```
Comment (rsanchez15) [inline include/databroker/communication/Bridge.hpp:60]: Too aggressive :joy:

File: include/databroker/communication/Bridge.hpp [hunk 2]
```diff
@@ -28,29 +30,86 @@ namespace eprosima {
...
 class Bridge
 {
 public:
 
+    /**
+     * Bridge constructor by required values
+     *
+     * In Bridge construction, the inside \c Tracks are created
+     *
+     * @param topic: Topic that this Bridge manage communication
```
Comment (rsanchez15) [inline include/databroker/communication/Bridge.hpp:49]:
```suggestion
     * @param topic: Topic of which this Bridge manages communication
```

File: include/databroker/communication/Bridge.hpp [hunk 3]
```diff
@@ -28,29 +30,86 @@ namespace eprosima {
...
+     * Does nothing if it is disabled
+     *
+     * Thread safe
+     */
+    void disable();
 
 protected:
 
+    /**
+     * Topic that this bridge manage communication
```
Comment (rsanchez15) [inline include/databroker/communication/Bridge.hpp:89]:
```suggestion
     * Topic of which this Bridge manages communication
```

**Review by rsanchez15 — CHANGES_REQUESTED** [review]

File: include/databroker/communication/Bridge.hpp (hunk 1 above)
Comment (jparisu) [reply include/databroker/communication/Bridge.hpp:60]: :boom:

File: include/databroker/communication/Bridge.hpp (hunk 2 above)
Comment (jparisu) [reply include/databroker/communication/Bridge.hpp:49]: :+1:

File: include/databroker/communication/Bridge.hpp (hunk 3 above)
Comment (jparisu) [reply include/databroker/communication/Bridge.hpp:89]: :+1:

**Review by rsanchez15 — APPROVED** [review]
LGTM

**Pull request merged**